test/object_store/test_backup.py

```diff
@@ -184,3 +186,62 @@ def list_sstables():
     print(f'Check that backup files are still there') # regression test for #20938
     post_objects = set([ o.key for o in get_s3_resource(s3_server).Bucket(s3_server.bucket_name).objects.filter(Prefix=prefix) ])
     assert objects == post_objects
+
+
+@pytest.mark.asyncio
+async def test_backup_and_one_to_one_restore(manager: ManagerClient, s3_server):
+    '''check that 1:1 restoring of a cluster works'''
+
+    cfg = {'enable_user_defined_functions': False,
+           'object_storage_config_file': str(s3_server.config_file),
+           'experimental_features': ['keyspace-storage-options'],
+           'task_ttl_in_seconds': 300
+           }
+    cmd = [ '--logger-log-level', 'sstables_loader=debug:sstable_directory=trace:snapshots=trace:s3=trace:sstable=debug:http=debug' ]
+    servers = await manager.servers_add(3, config=cfg, cmdline=cmd)
+
+    snap_name = unique_name('backup_')
+    logger.info(f'Create and backup keyspace (snapshot name is {snap_name})')
+
+    cql = manager.get_cql()
+    logger.info(f'Create keyspace')
+    ks, cf = create_ks_and_cf(cql, rf=3)
```

Comment:
this is too easy, to have 3 nodes and rf 3.
better stress the test with rf=1 to make sure data is properly backed up and restored.

Reply:
rf=1 over 3 nodes and rf=1 over {4 nodes in 2 racks} tests added